bundles/com.espressif.idf.core/src/com/espressif/idf/core/build/messages.properties

```diff
@@ -17,5 +17,5 @@ CMakeErrorParser_NotAWorkspaceResource=Could not map %s to a workspace resource.
 IDFBuildConfiguration_CMakeBuildConfiguration_NoToolchainFile=No toolchain file found
 IncreasePartitionSizeTitle=Low Application Partition Size
 IncreasePartitionSizeMessage=Less than 30% of application partition size is free({0} of {1} bytes), would you like to increase it? Please click <a href={2}>here</a> to check more details.
-ToolsInitializationDifferentPathMessageBoxTitle=Different IDF path found in the config file
-ToolsInitializationDifferentPathMessageBoxMessage=A different IDF path was found in the startup configuration file do you want to install the tools in this path or keep the older path\nNew Path: {0}\nOld Path: {1}
+ToolsInitializationDifferentPathMessageBoxTitle=Different IDF path found in IDE Environment
+ToolsInitializationDifferentPathMessageBoxMessage=Different IDF tools are installed in the IDE Environment: {0} than the one selected in startup config file for IDE (esp_idf.json).\nIf you want to keep the IDF tools in the IDE Environment select Yes(This will also add the tools to esp_idf.json).\nIf you want to revert to the last used tools in esp_idf.json press No.
```

Comment:
what about rephrasing it like this:
* 1 version:
The IDE Environment has different IDF tools installed compared to the ones selected in the startup-config file for the IDE (esp_idf.json). 
To keep the IDF tools {0} in the IDE Environment and update the tools in esp_idf.json, select "Yes."
To revert to the last used tools in esp_idf.json, press "No."


However, I found version 1 too verbose
* 2 version:
The IDF tools installed in the IDE Environment are different from the ones selected in the startup-config file (esp_idf.json) for the IDE. Do you want to keep the IDF tools {0} in the IDE Environment and update the tools in esp_idf.json?

what do you think?

Reply:
The second version is misleading to my understanding as I think it can confuse the user like they can press No thinking that the tools wont be added to json file but instead we are trying to revert the tools back to the original installation. This would be okay if we didnt have the usecase where the tools are updated via the IDE offline installer.